ci/rpm_build_unsuccessful.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Script to be run on unsuccessful RPM build
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ci_envs="$mydir/parse_ci_envs.sh"
+if [ -e "${ci_envs}" ]; then
+  # shellcheck disable=SC1091
```

Comment:
As noted elsewhere.

Reply:
As noted this is a limitation of using CentOS 7 for ShellCheck.